charts/stable/valheim/questions.yaml

```diff
@@ -9,205 +9,183 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: secretEnv
-    group: "App Configuration"
-    label: "Image Secrets"
+    group: App Configuration
+    label: Image Secrets
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: "SUPERVISOR_HTTP_USER"
+          label: Supervisor User
           schema:
             type: string
-            default: "admin"
+            default: ''
             required: true
         - variable: SUPERVISOR_HTTP_PASS
-          label: "SUPERVISOR_HTTP_PASS"
+          label: Supervisor PASS
           schema:
             type: string
             required: true
             private: true
-            default: "REPLACETHIS"
+            default: ''
```

Comment:
```suggestion
            default: ""
```

Reply:
Is there any kind of style guide or documentation that says "don't provide default credentials" or something along those lines?

Reply:
Nop, REPLACETHIS was added at somepoint to do some migrations, which would be failing without this.
But it's not needed.

I think it's better to have it empty so users dont blindly hit next and then looking around to find the username/password.